src/ResourceManagement/OperationalInsights/Microsoft.Azure.Management.OperationalInsights/Generated/Models/SearchResultsResponse.cs

```diff
@@ -0,0 +1,60 @@
+// Code generated by Microsoft (R) AutoRest Code Generator 1.0.0.0
+// Changes may cause incorrect behavior and will be lost if the code is
+// regenerated.
+
+namespace Microsoft.Azure.Management.OperationalInsights.Models
+{
+    using System.Linq;
+
+    /// <summary>
+    /// The get search result operation response.
+    /// </summary>
+    public partial class SearchResultsResponse
+    {
+        /// <summary>
+        /// Initializes a new instance of the SearchResultsResponse class.
+        /// </summary>
+        public SearchResultsResponse() { }
+
+        /// <summary>
+        /// Initializes a new instance of the SearchResultsResponse class.
+        /// </summary>
+        /// <param name="id">The id of the search, which includes the full
+        /// url.</param>
+        /// <param name="metadata">The metadata from search results.</param>
+        /// <param name="value">The array of result values.</param>
+        /// <param name="error">The error.</param>
+        public SearchResultsResponse(string id = default(string), SearchMetadata metadata = default(SearchMetadata), System.Collections.Generic.IList<object> value = default(System.Collections.Generic.IList<object>), SearchError error = default(SearchError))
+        {
+            Id = id;
+            Metadata = metadata;
+            Value = value;
+            Error = error;
+        }
+
+        /// <summary>
+        /// Gets the id of the search, which includes the full url.
+        /// </summary>
+        [Newtonsoft.Json.JsonProperty(PropertyName = "id")]
+        public string Id { get; private set; }
```

Comment:
Nope. This is just the results to a search. The id here uniquely identifies the search. The client will use the id to poll search results.